spec/beaker/dsl/install_utils/foss_utils_spec.rb

```diff
@@ -414,7 +414,7 @@ def logger
     context 'on el-5' do
       let(:platform) { Beaker::Platform.new('el-5-i386') }
       it 'installs' do
-        expect(subject).to receive(:on).with(hosts[0], /puppetlabs-release-el-5\.noarch\.rpm/)
+        expect(hosts[0]).to receive(:install_package_with_rpm).with('http://yum.puppetlabs.com/puppetlabs-release-el-5.noarch.rpm', '--replacepkgs', {:package_proxy=>false})
```

Comment:
But I have this line in install_puppetlabs_release_repo_on

    host.install_package_with_rpm(remote, '--replacepkgs', {:package_proxy => opts[:package_proxy]})

So the expectation is that the method under test is going to pass {:package_proxy => opts[:package_proxy]} and opts[:package_proxy] will be false.

Perhaps I should be more explicit by setting it to false before the test?